### PR TITLE
[Enhancement] Support syntax sugars file

### PR DESCRIPTION
**Pull request by Seaven**
## Problem Summary:
Fixes # (issue)

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 20 / 22 (90.91%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|6|7|85.71%|[5379]|
|:large_blue_circle:|com/starrocks/sql/parser/SyntaxSugars.java|14|15|93.33%|[31]|

**Review by kangkaisen — DISMISSED** [review]

**Review by LiShuMing — DISMISSED** [review]
LGTM~

File: fe/fe-core/src/main/java/com/starrocks/sql/parser/SyntaxSugars.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    /*
+     * function call syntax sugars
+     * - simple functions
+     * - aggregate functions
+     * - window functions
+     */
+    public static FunctionCallExpr parse(FunctionCallExpr call) {
+        if (FUNCTION_PARSER.containsKey(call.getFnName().getFunction())) {
```
Comment (packy92) [inline fe/fe-core/src/main/java/com/starrocks/sql/parser/SyntaxSugars.java:47]: use `getOrDeault` can remove this `if` branch

**Review by packy92 — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 18 / 20 (90.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|6|7|85.71%|[5379]|
|:large_blue_circle:|com/starrocks/sql/parser/SyntaxSugars.java|12|13|92.31%|[31]|

**Review by stdpain — APPROVED** [review]

**Pull request merged**